src/test/mir-opt/early_otherwise_branch_68867.rs

```diff
@@ -0,0 +1,32 @@
+// compile-flags: -Z mir-opt-level=3
+
+// example from #68867
+type CSSFloat = f32;
+
+pub enum ViewportPercentageLength {
+    Vw(CSSFloat),
+    Vh(CSSFloat),
+    Vmin(CSSFloat),
+    Vmax(CSSFloat),
+}
+
+// EMIT_MIR early_otherwise_branch_68867.try_sum.EarlyOtherwiseBranch.diff
+// EMIT_MIR early_otherwise_branch_68867.try_sum EarlyOtherwiseBranch.before SimplifyBranches-after-copy-prop.after
```

Comment:
@oli-obk Tidy complaines that this is longer than 100 chars. I'm not sure how to compress it. Any ideas?

Reply:
You add `// ignore-tidy-linelength` to this file

Reply:
Thanks, pushed that